src/main/java/impl/weather/DayTemperatureInfoImpl.java

```diff
@@ -0,0 +1,37 @@
+package impl.weather;
+
+import api.weather.DayTemperatureInfo;
+import kotlin.Pair;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.Month;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+
+public class DayTemperatureInfoImpl implements DayTemperatureInfo {
+    private int day;
```

Comment:
final

Reply:
What do you mean by final ??

Reply:
you can mark a field or variable as final, then link to this field won't be modified. It is good practice to mark such fields as final